[MRG] ENH: Demote format-conversion warnings to log messages (#1589)
mne-tools/mne-bids: doc/whats_new.rst

🧐 API and behavior changes
^^^^^^^^^^^^^^^^^^^^^^^^^^^

> - Format-conversion notices in :func:`mne_bids.write_raw_bids` no longer emit ``RuntimeWarning``\s when conversion is expected. Both ``Converting data files to <FORMAT> format`` and ``Encountered data in "<FMT>" format`` are now logged at INFO level. Adjust visibility with :func:`mne.set_log_level`, by `Bruno Aristimunha`_ (:gh:`1589`)

Reviewer: [nitpick] this is too much detail for a user-facing changelog entry IMO.

```suggestion
- Expected format conversions notices are now logged at ``info`` instead of ``warn`` level, by `Bruno Aristimunha`_ (:gh:`1589`)
```
